### ISSUE TITLE
What should probs be in Line 42 of actorcritic.py

### ISSUE THREAD

**Issue by namidairo777** [agakshat/maddpg#1]
`out = tf.contrib.distributions.RelaxedOneHotCategorical(0.1,probs=out)`

**Comment by agakshat** [comment]
Hi @namidairo777 : This repo is still a work in progress, I hope to have it running soon. About the  [RelaxedOneHotCategorical Distribution](https://www.tensorflow.org/api_docs/python/tf/contrib/distributions/RelaxedOneHotCategorical), probs is supposed to be the prior of each class in the Gumbel-Softmax estimator.

**Comment by namidairo777** [comment]
Sorry for the late reply! 
Thank you so much for answering me. 
I got a lot of inspirations of implementing MADDPG from your repo!

**Comment by agakshat** [comment]
Glad to hear it. I'm closing this issue for now.
